.github/workflows/build.ps1

```diff
@@ -8,5 +8,7 @@ $ErrorActionPreference = 'Stop'
 & ([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Channel 3.1 -InstallDir 'C:/Program Files/dotnet'
 dotnet user-secrets set --id f1506d66-289c-44cb-a2e2-80411cc690ec 'Plugin:Folder' '../../Plugin'
 dotnet user-secrets set --id f1506d66-289c-44cb-a2e2-80411cc690ed 'Plugin:Folder' '../../Plugin'
+dotnet user-secrets set --id f1506d66-289c-44cb-a2e2-80411cc690ec 'Plugin:Scripts:0' 'sk'
+dotnet user-secrets set --id f1506d66-289c-44cb-a2e2-80411cc690ed 'Plugin:Scripts:0' 'sk'
```

Comment:
if you're running this script as part of initdev aren't you installing a specific version of the SK extensions instead of using the one that was built as part of the github actions?

Reply:
@tbmeador-dlp this change is not required now since we are using grandbend.

Reply:
@tbmeador-dlp we are not moving sk.ps1 to Plugin folder. I'm setting the script to SK here just to turn off TPDM, so it will use default GrandBend template. Since there is no sk.ps1 in Implementation folder, it does nothing just to use the default template and it won't download the extension as well. We are also not specifying SK package in configuration.packages.json inside Implementation folder.